packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -27,35 +24,35 @@ interface DisclosureProps {
    * @default false
    */
   readonly defaultOpen?: boolean;
+
+  readonly onToggle?: (newOpened: boolean) => void;
+  readonly opened?: boolean;
 }
 
 export function Disclosure({
   children,
   title,
   defaultOpen = false,
+  onToggle,
+  opened,
```

Comment:
Prefer `open` rather than `opened` since opened is past tense and this is describing the current state.